### PR TITLE
Adding hometown file for githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
@hectorsector what kind of restaurants do you like so I can add them to the hometown file?

### CODE REVIEW

File: githubteacher-orlando.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Orlando, FL
+
+### Restaurants
```
Comment (hectorsector) [inline githubteacher-orlando.md:3]: I like [Pharmacy](http://thepharmacyorlando.com) in Dr. Philips and [Highball & Harvest](http://www.ritzcarlton.com/en/hotels/florida/orlando/dining/highball-and-harvest) at The Ritz.

**Review by hectorsector — CHANGES_REQUESTED** [review]

File: githubteacher-orlando.md (hunk 1 above)
Comment (hectorsector) [reply githubteacher-orlando.md:3]: I like 🍣, and deep dish 🍕

Comment (hectorsector) [reply githubteacher-orlando.md:3]: And 🍺? How about 🍻?

Comment (githubteacher) [reply githubteacher-orlando.md:3]: @hectorsector Added some restaurants, let me know this if this is good to :ship:

**Pull request merged**